<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="heading_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добро пожаловать в раздел ''Помощь''</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="answers_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>125</y>
      <width>841</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ответы на частозадаваемые вопросы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="first_question_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>202</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1. Как начать обучение?</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_answer_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ответ: Для того, чтобы начать обучение перейдите в  ''Учебник''-&gt;''Уроки'' и выберите интересующую эпоху</string>
    </property>
   </widget>
   <widget class="QLabel" name="second_question_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>322</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2. Как взаимодействовать с интерактивным календарем?</string>
    </property>
   </widget>
   <widget class="QLabel" name="second_answer_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ответ: Чтобы получить информацию о дате, вам нужно нажать на интересующую вас дату на самом календаре</string>
    </property>
   </widget>
   <widget class="QLabel" name="addition_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Помечание: в календаре присутствует информация только о значимых датах</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
